hygieia/.applier/group_vars/seed-hosts.yml

```diff
@@ -33,6 +34,18 @@ openshift_cluster_content:
       tags:
         - unpriv
         - all
+    - name: UI Builder Image
```

Comment:
@tomgeorge could you combine these two builds into a single template, consistent with the others?